src/context/ContextTypes.ts

```diff
@@ -734,34 +734,31 @@ export interface ChatSearchCriteria {
      *
      * Empty search criteria can be supported to allow resetting of filters.
      */
-    criteria: SearchCriteria[];
+    criteria: Array<OrganizationObject | string>;
```

Comment:
What would the string entail in this case?

Reply:
@psmulovics Any search string. Easiest to understand from the example here: https://fdc3.finos.org/docs/next/context/ref/ChatSearchCriteria#example

This could be described in the comment of course.

Not actually relevant to this PR, a previous PR missed regenerating these types so these changes are showing up on them all.